docs/reference/connector/apis/delete-connector-api.asciidoc

```diff
@@ -6,8 +6,7 @@
 
 beta::[]
 
-Removes a connector and associated sync jobs.
-This is a destructive action that is not recoverable.
+Soft-deletes a connector and removes associated sync jobs.
```

Comment:
Were you going to add a `?hard=<boolean>` query param?

Reply:
ahh yes! This should go in to docs as well! :)